### PR TITLE
fix: processing script with private identifier (closes DevExpress/testcafe#8067)

### PR DESCRIPTION
**Pull request by PavelMor25**
Added new condition to `property-get transformer` which excludes from transform the expression "object.#prop" into "__get$(object, '#prop')"

`ts-ignore` was added because updating `@types/estree` is required; the last update was three years ago. Updating now would require time for refactoring.

[esotope-hammerhead PR](https://github.com/DevExpress/esotope-hammerhead/pull/28)

### CODE REVIEW

**Review by AlexKamaev — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "bowser": "1.6.0",
     "crypto-md5": "^1.0.0",
     "debug": "4.3.1",
-    "esotope-hammerhead": "0.6.6",
+    "esotope-hammerhead": "https://github.com/PavelMor25/builds/raw/master/esotope-hammerhead-0.6.7.tgz",
```
Comment (Aleksey28) [inline package.json:24]: As I see your PR in `esotope-hammerhead` has already been merged, so you can publish a new version `esotope-hammerhead`, and set it here.

File: src/processing/script/transformers/property-get.ts [hunk 2]
```diff
@@ -32,6 +32,11 @@ const transformer: Transformer<MemberExpression> = {
         if (node.property.type === Syntax.Identifier && !shouldInstrumentProperty(node.property.name))
             return false;
 
+        // Skip: object.#prop
+        // @ts-ignore
+        if (node.property.type === Syntax.PrivateIdentifier)
```
Comment (Aleksey28) [inline src/processing/script/transformers/property-get.ts:37]: Have you tried to update  `estree`? Perhaps there won't be any problems.

**Review by Aleksey28 — CHANGES_REQUESTED** [review]
Please, add a link to the corresponding PR in `esotope-hammerhead`

File: src/processing/script/transformers/property-get.ts (hunk 2 above)
Comment (PavelMor25) [reply src/processing/script/transformers/property-get.ts:37]: Yes, I have tried, and this leads to numerous conflicts in TypeScript

Comment (Aleksey28) [reply src/processing/script/transformers/property-get.ts:37]: Okay, let's remove comment `// Skip: object.#prop` or refactor it in a way: `NOTE: <describing case>`

**Review by Aleksey28 — APPROVED** [review]

**Review by AlexKamaev — APPROVED** [review]

**Pull request merged**